app/brave_generated_resources.grd

```diff
@@ -166,6 +166,12 @@
       <message name="IDS_WALLET_PERMISSION_REQUEST_TEXT_FRAGMENT" desc="Text fragment for wallet installation permission request.">
           Install Crypto Wallets
       </message>
+      <message name="IDS_WIDEVINE_DONT_ASK_AGAIN_CHECKBOX" desc="Checkbox for prevent showing widevine install prompt">
+        Don't ask again
+      </message>
+      <message name="IDS_SETTINGS_ASK_WIDEVINE_INSTALL_DESC" desc="Text fragment for asking widevine install or not">
+        Ask when a site wants to install Widevine on your computer
```

Comment:
Minor nit to add a period "." at the end.  "Ask when a site wants to install Widevine on your computer."

Reply:
Done.